src/Raven.Server/Config/Categories/LogsConfiguration.cs

```diff
@@ -26,10 +26,23 @@ public class LogsConfiguration : ConfigurationCategory
         [ConfigurationEntry("Logs.MaxFileSizeInMb", ConfigurationEntryScope.ServerWideOnly)]
         public Size MaxFileSize { get; set; }
 
-        [Description("How far back we should retain log entries")]
-        [DefaultValue(3 * 24)]
-        [TimeUnit(TimeUnit.Hours)]
-        [ConfigurationEntry("Logs.RetentionTimeInHrs", ConfigurationEntryScope.ServerWideOnly)]
-        public TimeSetting RetentionTime { get; set; }
+        [Description("How far back we should retain log entries in days")]
+        [DefaultValue(null)]
+        [MinValue(1)]
+        [TimeUnit(TimeUnit.Days)]
+        [ConfigurationEntry("Logs.RetentionTimeInDays", ConfigurationEntryScope.ServerWideOnly)]
+        public TimeSetting? RetentionTime { get; set; }
+
+        [Description("The maximum size of the log after which the old files will be deleted")]
+        [DefaultValue(null)]
+        [MinValue(256)]
```

Comment:
Just a sanity check here... having this attribute and providing `null` will not set the min value, right?

Reply:
Do you mean if I can set the value below 256 because of the `DefaultValue` is set to null?

Reply:
No, what will happen when I will set the value to `null` explicitly in `settings.json` ?